src/app/components/FrostedGlassPromo/index.jsx

```diff
@@ -117,24 +143,26 @@ const FrostedGlassPromo = ({
           image,
         )}
       />
-      <FrostedGlassPanel
-        image={image.src}
-        minimumContrast={minimumContrast}
-        paletteSize={paletteSize}
-      >
-        <H3>
-          <A
-            script={script}
-            service={service}
-            href={url}
-            onClick={onClick}
+      <Lazyload
+        offset={PANEL_OFFSET}
```

Comment:
Not familiar with our lazyload component, what does this offset do?

Reply:
It allows the component to render when its 250px from being inside the viewport. Just helps prevent that obvious flash when a component is loaded, since it's loaded offscreen just before it comes into view.